cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "68b291ad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from matplotlib import pyplot as plt\n",
+    "from datetime import datetime\n",
+    "\n",
+    "from utils.model_training import train_model\n",
+    "from utils.model_training import sum_xval_folds\n",
+    "from cleanlab.multiannotator import get_majority_vote_label\n",
+    "from cleanlab.multiannotator import get_label_quality_multiannotator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2f3551b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "now = datetime.now() # Current date and time\n",
+    "experiment_path = \"./experiment_\" + str(int(now.timestamp()))\n",
+    "\n",
+    "if not os.path.exists(experiment_path):\n",
+    "    os.makedirs(experiment_path)\n",
+    "    print(\"Directory \" , experiment_path ,  \" Created \")\n",
+    "else:    \n",
+    "    print(\"Directory \" , experiment_path ,  \" already exists\")\n",
+    "\n",
+    "print(f'Experiment saved in {experiment_path}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "23c50596",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import cifar10 data for model training or download it yourself by calling cifar2png cifar10 ./data/cifar10 --name-with-batch-index\n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/Multiannotator/cifar-10/cifar10_test.tar.gz'\n",
+    "!tar -xzf cifar10_test.tar.gz\n",
+    "\n",
+    "# Import cifar10h pre calculated multiannotator labels and image paths\n",
+    "!cd $experiment_path && wget -nc 'https://cleanlab-public.s3.amazonaws.com/Multiannotator/cifar-10h/cifar-10h-worst25-coin20/c10h_labels_worst25_coin20.npy'\n",
+    "!cd $experiment_path && wget -nc 'https://cleanlab-public.s3.amazonaws.com/Multiannotator/cifar-10h/cifar-10h-worst25-coin20/c10h_image_paths.npy'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d2bed92",
+   "metadata": {},
+   "source": [
+    "## 2. Load multiannotator labels and generate consensus labels for them\n",
+    "`multiannotator_labels` for this example is a precalculated subset of the original `cifar10h` annotator labels. The subset takes the worst 25 annotators and the incrementally add annotators from worst to best if they share annotations with those already in the subset until each of the 10,000 examples has at least 1 annotation. \n",
+    "\n",
+    "The reason for this being `cifar10` is an unnaturally easy dataset to label and using the original dataset would make the annotator agreement too high for our method to contribute meaningful improvement. Additionally in practice it is rare to have 50 annotators annotate a specific example but this subset ensures significantly sparser annotations."
```

Comment:
Let's only keep the first sentence in this notebook. The user of this example does not need to understand this dataset beyond that each example has been labeled by multiple annotators. 

Save that nice text for our write-up though!  There we will need to justify why we used this variant of cifar10h